builder/virtualbox/common/step_type_boot_command.go

```diff
@@ -82,6 +82,8 @@ func (s *StepTypeBootCommand) Run(ctx context.Context, state multistep.StateBag)
 
 	ui.Say("Typing the boot command...")
 	command, err := interpolate.Render(s.BootCommand, &s.Ctx)
+	// Second interpolation
+	command, err = interpolate.Render(command, &s.Ctx)
```

Comment:
If this is something we want to keep, I thought on check if this is a user variable and from that decide if a second interpolation is necessary. We could check the string if contains `{{ user ... }}`